app/views/guide_icons_images.html

```diff
@@ -70,7 +70,7 @@ <h3 class="heading-medium" id="images">Images</h3>
         <img src="/public/images/examples/3by2.jpg" alt="3:2">
       </div>
       <div class="column-one-half">
-        <img src="/public/images/examples/pm.jpg" alt="">
```

Comment:
Might be used in prototypes so removing it would potentially be a breaking change. Might as well leave it in for the time being.